src/GraphQLParser.ApiTests/GraphQL-Parser.approved.txt

```diff
@@ -142,7 +143,7 @@ namespace GraphQLParser.AST
         public System.Collections.Generic.List<GraphQLParser.AST.GraphQLDirective>? Directives { get; set; }
         public override GraphQLParser.AST.ASTNodeKind Kind { get; }
         public GraphQLParser.AST.GraphQLSelectionSet? SelectionSet { get; set; }
-        public GraphQLParser.AST.GraphQLNamedType? TypeCondition { get; set; }
```

Comment:
see http://spec.graphql.org/October2021/#sec-Inline-Fragments and http://spec.graphql.org/October2021/#FragmentDefinition  - [TypeCondition](http://spec.graphql.org/October2021/#TypeCondition) node should be added